docs/pmm-upgrade/upgrade_agent.md

```diff
@@ -1,10 +1,46 @@
-# Upgrade PMM agent
+# Upgrade PMM Client
 
-PMM-Agent can be updated from tarball:
+There are two primary methods to update PMM Clients, depending on your initial installation method:
 {.power-number}
 
- 1. Download `tar.gz` with `pmm2-client`.
- 2. Extract it.
+1. Using your operating system's package manager.
+2. Updating from a tarball.
+
+### 1. Package Manager method
+
+The package manager method is generally more convenient and efficient. Percona provides the [percona-release](https://docs.percona.com/percona-software-repositories/installing.html) package, which helps you install Percona software, including the PMM Client. The PMM Client is available from the `pmm-client` repository.
```

Comment:
```suggestion
The package manager method is generally more convenient and efficient. Percona provides the [percona-release](https://docs.percona.com/percona-software-repositories/installing.html) package, which helps you install Percona software, including PMM Client. PMM Client is available from the `pmm-client` repository.
```

Reply:
This is a product name, so it should come without the article IMO. Similar to Docker, for example, or Nomad.